scripts/encode_data.py

```diff
@@ -17,16 +17,21 @@
 
```

Comment:
It looks like you added the changes from the parent_docs commit here. You should remove the query changes from that pr and remove the parent_docs changes from here.

Reply:
I think I removed the parent_docs changes from this pr. Can you review this pr?